codalab/requirements/common.txt

```diff
@@ -1,70 +1,74 @@
-Django==1.6.11
+Django==1.11.29
 alembic==0.6.7
 anyjson==0.3.3
 argparse
 # azure==0.7.1
 closure_linter
-django-analytical==0.15.0
-django-appconf==0.6
-django-compressor==1.3
-django-config-gen==1.0.8
-# django-configurations
-django-configurations==0.8
-django-debug-toolbar==1.3.0
-sqlparse==0.1.19
+django-analytical==2.0.0
+django-appconf==1.0
+django-compressor==2.2
+django-configurations==2.0
+django-debug-toolbar==2.2
+sqlparse==0.2.0
 django-debug-panel==0.8.1
-django-extensions==1.5.7
+django-extensions==1.7.6
 django-extra-views==0.6.3
-django-filter==0.7
-django-guardian==1.1.1
-django-haystack==2.4.0
+django-filter==1.1.0
+django-guardian==1.4.9
+django-haystack==2.8.1
 django-js-reverse>=0.2.0
-django-mptt==0.5.5
-django-nose==1.4.1
-django-oauth-toolkit==0.6.1
-#django-storages==1.1.8 # Replaced by storages-redux.. need to uninstall storages..
-django-storages-redux==1.3.2
-django-tinymce==2.0.5
-djangorestframework==2.3.6
+django-mptt==0.9.1
+django-nose==1.4.6
+django-oauth-toolkit==0.11.0
+# django-storages==1.1.8 # Replaced by storages-redux.. need to uninstall storages..
+# Regular django-storages is now maintained by the author of redux
+django-storages==1.9.1
+django-tinymce==2.7.1
+djangorestframework==3.8.2
 mock==1.0.1
-oauthlib==0.6.1
+oauthlib==1.1.2
 psutil==2.1.1
 python-openid==2.2.5
 python-ptrace==0.6.5
-pytz==2013b
-PyYAML==3.10
+pytz==2015.7
+PyYAML==5.1
 requests-oauthlib==0.3.2
-requests==1.2.3
+requests==2.24.0
 six==1.10.0
-South==1.0.2
 SQLAlchemy==0.9.7
 watchdog==0.8.1
-wsgiref==0.1.2
-django-simple-captcha==0.4.5
-django-bootstrap-form==3.2
+#wsgiref==0.1.2
+django-simple-captcha==0.5.12
+django-bootstrap-form==3.4
 newrelic==2.54.0.41
 pylibmc==1.5.2
 
-celery==3.1.26.post2
+celery==4.4.6
 boto==2.45.0
```

Comment:
why install boto and boto3? just do 1?

Reply:
I figured we'd still leave boto compatible and leave it up to the user for which back-end to use. If we just do boto3 there's probably a lot that can be removed/changed.

Reply:
What is using boto?

Reply:
You can still use the old boto backend with Django-storages. Shouldn't have to change any settings in regards to storage when updating.

Reply:
Can we pull it?

Reply:
Pulled